CMakeLists.txt

```diff
@@ -152,46 +152,24 @@ if (ENABLE_CODECOVERAGE)
 endif (ENABLE_CODECOVERAGE)
 
 # Search paths for Postgres binaries
-if (WIN32)
+if(WIN32)
   find_path(PG_PATH
-    bin/postgres
-    HINTS
-    "C:/PostgreSQL"
-    "C:/Program Files/PostgreSQL"
-    PATH_SUFFIXES
-    bin
-    10/bin
-    96/bin
-    pg96/bin
-    DOC
-    "The path to a PostgreSQL installation")
-endif (WIN32)
-
-if (UNIX)
+    postgres.exe
+    PATHS "C:/PostgreSQL" "C:/Program Files/PostgreSQL"
+    PATH_SUFFIXES bin 10/bin 96/bin pg96/bin
```

Comment:
Without specifying a proper version number directory here, the build still fails for me.

I think we need to add `11/bin` and `12/bin` as well

Reply:
Ah, will do.

Reply:
Done, but since we do not support PG12 yet, I didn't add that. Will add it once the PG12 branch is merged.